msal/default_web_request_manager.go

```diff
@@ -246,11 +246,11 @@ const (
 )
 
 func addAADHeaders(headers http.Header, authParameters msalbase.AuthParametersInternal) {
```

Comment:
guessing this gets removed with the business logic changes?

Reply:
These won't get removed until the integration changes.  The business logic will be separate, but I don't want to integrate at the same time because that will probably have massive impacts.